measure/performance/MSPerf.py

```diff
@@ -137,7 +137,7 @@ def main():
     ################################# Variable Assignment #################################################
     starting_streamnumber           = str(ARGS.startStreams) if ARGS.startStreams else "all:1"
     maximum_iteration               = int(ARGS.numbers_of_iteration) if ARGS.numbers_of_iteration else 1
-    endStreams             = int(ARGS.endStreams) if ARGS.endStreams else 99
+    endStreams                      = int(ARGS.endStreams) if ARGS.endStreams else 999999
```

Comment:
I think you are setting the default in a wrong place (and for other options as well!). Instead you need to do that during option declarations. I.e.:

```
parser.add_argument('-e', '--endStreams', default=999999, help='To set ending number of multi stream performance measurement')

<...>

endStreams                      = ARGS.endStreams
```

Could you, please, rework options declarations in general taking above into account?